document Slack thread mirroring for Case Management (#37018)
DataDog/documentation: content/en/incident_response/case_management/notifications_integrations.md

When enabled, any case that meets your specified query criteria (at any point in its lifecycle) automatically triggers an incident, enabling faster response times for your team.

## Slack mirroring
With the Slack integration, replies in Slack notification threads linked to a case are automatically mirrored to the case activity timeline. This keeps the case context up to date without requiring manual updates in Datadog. Slack thread mirroring to cases is supported for:
- [Slack notifications][8] generated from Case Management
- Slack notifications generated from Monitors using [case handles][7]
- Slack threads for cases created directly from Slack using the [Slack integration][9]

**To configure Slack thread mirroring**:

> Ensure the [Slack integration][9] is configured for your Datadog organization.

Reviewer: ```suggestion
Make sure the [Slack integration][9] is configured for your Datadog organization.
```

Slack thread mirroring is enabled by default for all Case Management projects. To disable it for a specific project:

> 1. Go to [**Project Settings**][1] and click on a project to expand its settings.

Reviewer: ```suggestion
1. Navigate to [**Project Settings**][1] and click on a project to expand its settings.
```

1. In the expanded menu, click **Integrations** > **Slack**.

> 1. Toggle off **Slack thread mirroring**

Reviewer: ```suggestion
1. Toggle off **Slack thread mirroring**.
```

> ### How It Works

Reviewer: ```suggestion
### How it works
```
